packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,212 @@
+| value                                                                                              | `string`                          | No                                                                                  |                                                           | Makes input field [controlled](https://reactjs.org/docs/forms.html#controlled-components)                                                                                                                                                                                             |
+| onChange                                                                                           | `({ event, name, value }) => {}`  | No                                                                                  |                                                           | Function called when the value of the input field changes                                                                                                                                                                                                                             |
+| onBlur                                                                                             | `({ event, name, value }) => {}`  | No                                                                                  |                                                           | Function called when the input field loses focus                                                                                                                                                                                                                                      |
+| isDisabled                                                                                         | `boolean`                         | No                                                                                  | `false`                                                   | Marks the input field as disabled. It'll also loose keyboard focus.                                                                                                                                                                                                                   |
+| showClearButton                                                                                    | `boolean`                         | No                                                                                  | `false`                                                   | Defines if we want to show the clear icon button on the right side of the password field                                                                                                                                                                                              |
```

Comment:
Should we be showing a clear button on the password field? I think we already have the eye icon towards the right for our password fields, adding an extra icon would make it a little messy. I don't see the clear button in our designs though.

Reply:
clear button will only be shown when the input field is in active state so i think should be fine vs not providing it at all which is valid case for all the input fields